python/py-setuptools_scm_git_archive/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-setuptools_scm_git_archive
+version             1.0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+master_sites        pypi:s/setuptools_scm_git_archive
+distname            setuptools_scm_git_archive-${version}
+
+python.versions     27 34 35 36 37
```

Comment:
New ports should not add 3.4 or 3.5, both are obsolete.

Reply:
Fixed in the new commit